code/modules/mob/living/basic/space_fauna/spider/spider_abilities/hivemind.dm

```diff
@@ -61,10 +61,36 @@
 /datum/action/command_spiders/proc/spider_command(mob/living/user, message)
 	if(!message)
 		return
-	var/my_message = span_spider("<b>Command from [user]:</b> [message]")
+	var/my_message = format_message(user,message)
 	for(var/mob/living/basic/spider as anything in GLOB.spidermobs)
 		to_chat(spider, my_message)
 	for(var/ghost in GLOB.dead_mob_list)
 		var/link = FOLLOW_LINK(ghost, user)
 		to_chat(ghost, "[link] [my_message]")
 	user.log_talk(message, LOG_SAY, tag = "spider command")
+
+/datum/action/command_spiders/proc/format_message(mob/living/user, message)
+	return span_spiderbroodmother(("<b>Command from [user]:</b> [message]"))
+
+/**
+ * Sends a small message to all currently living spiders.
+ */
+/datum/action/command_spiders/communication_spiders
+	name = "Communication"
+	desc = "Send a command to all living spiders."
```

Comment:
Is this true? I thought the point was that this one _wasn't_ a command

Reply:
yeah thats just a typo